apps/src/util/sort.js

```diff
@@ -0,0 +1,14 @@
+// Sorts an array of objects by name, case insensitive
```

Comment:
should this comment be updated since it's a comparator, it doesn't operate on an array?

Reply:
Good call!